src/Metadata/Property/PropertyMetadata.php

```diff
@@ -43,6 +44,7 @@ public function __construct(Type $type = null, string $description = null, bool
         $this->identifier = $identifier;
         $this->iri = $iri;
         $this->attributes = $attributes;
+        $this->childInherited = null;
```

Comment:
Why isn't it possible to set it through the constructor (when you use API Platform as a component set)?